### PR TITLE
[LLM Runtime] Update BGE README.md

### PR DESCRIPTION
**Pull request by a32543254**
## Type of Change

feature or bug fix or documentation or others
API changed or not

## Description

detail description 
JIRA ticket: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

File: examples/huggingface/pytorch/text-embedding/deployment/mteb/bge/README.md [hunk 1]
```diff
@@ -65,13 +65,29 @@ bash run_bge.sh --model=BAAI/bge-base-en-v1.5 --precision=dynamic_int8 --mode=th
...
+tokenizer = AutoTokenizer.from_pretrained('BAAI/bge-base-en-v1.5')
+encoded_input = tokenizer(sentences_batch,
+                            padding=True,
+                            truncation=True,
+                            max_length=512,
+                            return_tensors="np")
+
+engine_input = [encoded_input['input_ids'], encoded_input['token_type_ids'], encoded_input['attention_mask']]
+
+model = AutoModel.from_pretrained('./model_and_tokenizer/int8-model.onnx', use_embedding_runtime=True)
```
[the remark this answers is not in the document]
Comment (a32543254) [reply examples/huggingface/pytorch/text-embedding/deployment/mteb/bge/README.md:84]: the HF link is available only when the model has pass the PDT process.

Comment (kevinintel) [inline examples/huggingface/pytorch/text-embedding/deployment/mteb/bge/README.md:84]: It's better to list supported data type of use_embedding_runtime=True

Comment (hshen14) [reply examples/huggingface/pytorch/text-embedding/deployment/mteb/bge/README.md:84]: supported data types?

Comment (kevinintel) [reply examples/huggingface/pytorch/text-embedding/deployment/mteb/bge/README.md:84]: some users may want to use load_in_4bit=True

Comment (hshen14) [reply examples/huggingface/pytorch/text-embedding/deployment/mteb/bge/README.md:84]: it makes sense to add a table to show what data types LLM runtime supports for BGE models.

**Review by Zhenzhong1 — APPROVED** [review]

**Review by hshen14 — APPROVED** [review]

**Pull request merged**